mars-core/src/test/java/com/mars/core/util/FileUtilTests.java

```diff
@@ -0,0 +1,120 @@
+package com.mars.core.util;
+
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import com.alibaba.fastjson.JSON;
+import org.ho.yaml.Yaml;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+import org.junit.rules.ExpectedException;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.awt.image.BufferedImage;
+import java.io.*;
+import java.lang.reflect.Method;
+import javax.imageio.ImageIO;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({"javax.management.*"})
+public class FileUtilTests {
+
+    @PrepareForTest({FileUtil.class})
+    @Test
+    public void testReadFileString1() throws Exception {
+        InputStreamReader inputStreamReader =
+                PowerMockito.mock(InputStreamReader.class);
+        BufferedReader bufferedReader =
+                new BufferedReader(new StringReader("line1\nline2\nline3"));
+        PowerMockito.whenNew(InputStreamReader.class).withAnyArguments()
+                .thenReturn(inputStreamReader);
+        PowerMockito.whenNew(BufferedReader.class).withAnyArguments()
+                .thenReturn(bufferedReader);
+
+        Assert.assertEquals("line1line2line3", FileUtil.readFileString("foo"));
+    }
+
+    @Test
+    public void testReadFileString2() {
+        FileUtil.readFileString(null);
+    }
+
+    @PrepareForTest({JSON.class, Yaml.class, FileUtil.class})
+    @Test
+    public void testReadYml() throws Exception {
+        PowerMockito.mockStatic(Yaml.class);
+        PowerMockito.mockStatic(JSON.class);
+
+        Method toJSONStringMethod = DTUMemberMatcher.method(
+                JSON.class, "toJSONString", Object.class);
+        PowerMockito.doReturn("foo")
+                .when(JSON.class, toJSONStringMethod)
+                .withArguments(or(isA(Object.class), isNull(Object.class)));
+
+        Assert.assertEquals("foo", FileUtil.readYml("a/b/c"));
+    }
+
+    @Rule
+    public final ExpectedException thrown = ExpectedException.none();
```

Comment:
I'd put this at the top of the class, before any tests

Reply:
Done